extract/README.md

```diff
@@ -0,0 +1,126 @@
+# caliper-event-analysis: Extract Events From the LRS' DB
+
+## Open an SSH tunnel to the LRS' DB _(optional)_ 
+
+If the LRS' MongoDB service is on a protected network (e.g., behind a 
+firewall), use SSH to connect to its server and open a tunnel to the
+service to make it appear to be available locally.
+
+1. `cp tunnel-example.sh tunnel-.sh`
+1. Edit `tunnel-.sh`
+    1. Change `remote-user` to the username on the server.
+    1. Change `remote-host.example.org` to the hostname of the server.
+1. Run `tunnel-.sh` to set up the tunnel
+    1. Consider the requirements for connecting to the server
+        specified in the script before running it.
+        (E.g., you may need to use a VPN.)
+    1. The tunnel will make the remote service appear to be local,
+        responding to the address `127.0.0.1:27017/remote-db-name`.
+    1. The script displays verbose output to allow monitoring the
+        connection and to make it easy to close the tunnel later.
+    1. Since this script occupies a terminal, it's best to run the
+        following commands in another terminal.
+        
+## Set the LRS' DB address in the environment _(optional)_ 
+
+The `extract` program can be given the address of the LRS' DB using
+a command line option.  However, it may be more convenient to set
+the address in the `MONGODB_ADDRESS` environment variable.  This is
+especially true if `extract` will be run interactively rather than as
+a batch process.
+
+1. `cp env-example.sh env-.sh`
+1. Edit `env-.sh`
+    1. Change `remote-db-name` to the name of the LRS' MongoDB on the
+        server.
+1. Run `source env-.sh` to set up the tunnel
+    1. Using the `source` command is required to set the environment
+        variable in the current shell.
```

Comment:
I think it will be useful to add a line in here to verify the `MONGODB_ADDRESS` envi variable is set as expected using `printenv`